all.js

```diff
@@ -25,9 +25,26 @@ function uploadFilesViaScp(sourceFileList, targetFileList, cb) {
     return;
   }
 
-  var prefix = config.device_user_name + ':' + config.device_password + '@' + config.device_host_name_or_ip_address + ':';
+  let scpOptions = {
```

Comment:
Most, var vs. const/let are two different variable declaration. Either use var uniquely or const/let combined. It is not suggested to mix the two coding styles.